system/inc/system_version.h

```diff
@@ -191,6 +191,8 @@ extern "C" {
 #define SYSTEM_VERSION_v400ALPHA1   SYSTEM_VERSION_ALPHA(4, 0, 0, 1)
 #define SYSTEM_VERSION_v400ALPHA2   SYSTEM_VERSION_ALPHA(4, 0, 0, 2)
 #define SYSTEM_VERSION_v400BETA1   SYSTEM_VERSION_BETA(4, 0, 0, 1)
+#define SYSTEM_VERSION_v400         SYSTEM_VERSION_DEFAULT(4, 0, 0)
+// These are required for some of the dynalib checks
```

Comment:
Do we need this comment, and in this location?

Reply:
I dont know, it was in the 4.0.0 commit. It seems it refers to 5.0.0 alpha 1 and 2 versions, not sure if its relevant anymore

Reply:
Yeah we can safely remove that 👍🏼